backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/ResourceAdministrationPoint.cs

```diff
@@ -154,23 +177,110 @@ private async Task<List<ServiceResource>> GetResources()
             return resources;
         }
 
+        private async Task<List<ServiceResource>> GetFullResourceList()
+        {
+            string cacheKey = $"resources:fulllist";
+            if (!_memoryCache.TryGetValue(cacheKey, out List<ServiceResource> resources))
+            {
+                // TODO: Rewrite current GetResurces to use new API instead of search?
```

Comment:
Well, it was actually a question aimed more at the other GetResources (the one for maskinportenSchemas), since that one currently uses search.
But I think I've decided to just leave it as-is for the time being so this todo should probably be removed